CMakeLists.txt

```diff
@@ -1,5 +1,19 @@
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
 
+# Set up vcpkg by default. This can be overridden on the command-line, or
+# edited after configuration.
+set(CMAKE_TOOLCHAIN_FILE
+    "${CMAKE_CURRENT_SOURCE_DIR}/vendor/vcpkg/scripts/buildsystems/vcpkg.cmake"
+)
+set(VCPKG_BOOTSTRAP_OPTIONS "-disableMetrics")
```

Comment:
What are these metrics?

Reply:
From https://learn.microsoft.com/en-us/vcpkg/about/privacy#data-collected:

> ... telemetry events such as the **command line used**, the **time of invocation**, and **how long execution took**. Some commands also add additional calculated information (such as the **full set of libraries to install**) ...

Emphasis mine.

Reply:
Yikes, Microsoft is creepy.